MM/5.9.0/config/media_manager_5/labels/Administration.dcl

```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
+
+resource configservice_label administration_tools_dashboard_tool_generate_member_logins {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_GENERATE_MEMBER_LOGINS'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Generate user logins'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Lav bruger-logins'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_tool_generate_asset_downloads {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_GENERATE_ASSET_DOWNLOADS'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Generate asset downloads'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Lav asset-downloads'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_tool_unsaved {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_UNSAVED'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Unsaved'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Ikke gemt'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_tool_new {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_NEW'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'New'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Ny'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_block_edit {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_BLOCK_EDIT'
+  group = 'administration-tools - dashboard-block
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Edit'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Rediger'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label administration_tools_analytics_block_builder_time_bucket_size {
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_BLOCK_BUILDER_TIME_BUCKET_SIZE'
+  group = 'administration-tools - analytics-block-builder
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Time bucket size'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tids-spands-størrelse'
```

Comment:
```suggestion
      default_translation = 'Tid-spands-størrelse'
```

Reply:
I was unsure of which one was most correct. So, I approved this. You may choose to add this suggestion.

Reply:
Honestly, both kinda sucks, i have no idea what you would even call this thing in danish :D